client/index.html

```diff
@@ -10,10 +10,11 @@
       crossorigin="anonymous"
     />
     <title>Kite 🪁</title>
-    <!-- <link rel="stylesheet" href="/style.css" /> -->
-    <script defer src="/bundle.js"></script>
+    <!-- <link rel="stylesheet" href="./style.css" /> -->
+    <script defer src="./index.js"></script>
```

Comment:
One key change is parcel will rewrite this `index.html` file. 

In webpack we have to get this src path to be the right one for a web browser. In parcel we instead reference the path more like if we were using `require` or `import`.